src/tasks/WasmAppBuilder/IcallTableGenerator.cs

```diff
@@ -203,7 +203,7 @@ private void ProcessType(Type type)
                 }
                 catch (NotImplementedException nie)
                 {
-                    Log.LogWarning($"Failed to generate icall function for method '[{method.DeclaringType!.Assembly.GetName().Name}] {className}::{method.Name}'" +
+                    Log.LogWarning(null, "WASM0001", "", "", 0, 0, 0, 0, $"Failed to generate icall function for method '[{method.DeclaringType!.Assembly.GetName().Name}] {className}::{method.Name}'" +
```

Comment:
What are the empty strings and zeros for?

Reply:
It's the only overload of `LogWarning` with "warning code" this I found, or did I missed any?

Reply:
@maraf @ilonatommy this question needs to be addressed before I merge.

Reply:
This is fine.
On `main` though, or for future similar change we should use named parameters.

Reply:
> What are the empty strings and zeros for?

https://learn.microsoft.com/en-us/dotnet/api/microsoft.build.utilities.tasklogginghelper.logwarning?view=msbuild-17-netcore#microsoft-build-utilities-tasklogginghelper-logwarning(system-string-system-string-system-string-system-string-system-int32-system-int32-system-int32-system-int32-system-string-system-object())

Reply:
But even with named parameters, we would need to specify all those zeros and empty strings, right?